test/api/User.js

```diff
@@ -716,4 +716,204 @@ describe('User', () => {
       });
     });
   });
+
+  describe('POST /shortcutsearchusers', () => {
+    const queryUser = { query: 'coOl' };
+    const fiveMatchUsers = { query: 'Lot' };
+    const url = '/api/user/shortcutsearchusers';
+    const requiredLevel = MEMBERSHIP_STATE.OFFICER;
+    let accessLevel = '';
+
+    it('Should return status code 403 if no token is passed through', async () => {
+      setTokenStatus(false);
+      const result = await test.sendPostRequest(url, queryUser);
+      expect(result).to.have.status(FORBIDDEN);
+    });
+
+    it('Should return status code 401 if access level is invalid', async () => {
+      accessLevel = MEMBERSHIP_STATE.MEMBER;
+      setTokenStatus(accessLevel >= requiredLevel, {accessLevel});
```

Comment:
each line with `accessLevel >= requiredLevel`, please hardcode it to true/false, it will read much easier